ArmPkg/Drivers/CpuDxe/PageTableMemoryAllocation.c

```diff
@@ -104,6 +104,7 @@ FindPoolToAllocateFrom (
   @retval A pointer to the allocated page table memory or NULL if the
           allocation failed.
 **/
+STATIC

```

Comment:
```suggestion
static
```
https://edk2.groups.io/g/devel/topic/110977230#msg121076